### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by takayukioda**
Made release note

### CODE REVIEW

**Comment by takayukioda** [comment]
@shunjikonishi please review me!

File: source/docs/ja/release/notes.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+応募者管理システムの MyPage にリンクを置く等、配信コストを減らすことが可能になります。
+
+### エントリーシート機能を試験にも追加しました
+イベントだけでなく、試験においても本人確認に必要な情報を入力してから受験してもらう事が可能になりました。  
+試験ごとに自由に設定が可能なので、応募者管理システムのIDなども取得が可能です。
+
+## 機能改善
+
+### 表示件数の変更機能を追加しました
+今までは 10 件ずつで固定だった表示件数ですが、この修正で最大50件までを一度に表示させる事ができるようになりました。
```
Comment (shunjikonishi) [inline source/docs/ja/release/notes.md]: どこの？

**Review by shunjikonishi — COMMENTED** [review]
@takayukioda fix me
I think it is better to separate each release not files.

For mkdoc, can you use `releases/index.md`?

**Comment by shunjikonishi** [comment]
@takayukioda fix me

**Comment by takayukioda** [comment]
@shunjikonishi please review me
update the structures (please ignore the middle changes and see final update; I change structures a lot)

**Comment by shunjikonishi** [comment]
LGTM

**Pull request merged**